### DISCUSSION TITLE
Welcome to tasker-guide Improvement Discussions

### DISCUSSION THREAD

**Discussion by jcasman** [theta360developers/tasker-guide#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jcasman** [comment]
This Tasker Guide is a test for working on the larger theta-client SDK document. We are interested in managing the project through GitHub. 

Here is the current workflow:

- Erik creates his own branch
- Erik edits his branch
- Erik creates pull request
- Jesse/Craig review and test
- Jesse/Craig create issues, if needed. Issues are specific and concise. If solvable, please solve and response through GitHub. If you have questions, working through GitHub comments on issues is best. If not, email, chat or even discussions in the office are fine.
- - Erik gets notification through GitHub (email)
- Jesse/Craig merge Pull Request
- Jesse/Craig add Discussion topics, if needed. Discussion topics are intended to be larger and more strategic. Like this one, laying out the workflow. Other Discussion topics may be upcoming assignments. Or discussions how to better improve the theta-client guide overall.

**Reply by AirTechWick** [reply]
Hello, should I create a branch for each issue and then submit or just create a branch for the issues of the day that I'm working on?

**Comment by jcasman** [comment]
Create a branch for the issues of the day. No need to be more granular than that.

**Comment by codetricity** [comment]
implement conventional commits
https://www.conventionalcommits.org/en/v1.0.0/

Use `feat: ` and `fix: `. 
Use multi-line commits
